anw678(TheDescent)/TheDescentPart2.py

```diff
@@ -0,0 +1,110 @@
+# The Descent
+# ====================
+#
+#
+#
+# This documentation helps solve "The Descent" on `codingame.com <https://www.codingame.com/ide/puzzle/the-descent>`_.
+#
+# By Austin Wesley
+#
+# The Problem
+# ----------------
+# .. image:: descent.jpg
+#   :width: 100%
+#
+# Destroy the mountains before your starship collides with one of them. For that, shoot the highest mountain on your path.
+# At the start of each game turn, you are given the height of the 8 mountains from left to right.
+# By the end of the game turn, you must fire on the highest mountain by outputting its index (from 0 to 7).
+#
+# Firing on a mountain will only destroy part of it, reducing its height. Your ship descends after each pass.
+#
+# -----------------------------------------------------------------
+#
+# * The while loop represents the game.
+# * Each iteration represents a turn of the game where you are given inputs
+#   (the heights of the mountains)and where you have to print an output (the index of the mountain to fire on)
+# * The inputs you are given are automatically updated according to your last actions.
+#
+#
+#
+#
+# The  Game Loop
+# __________________
+# The solution method for this problem is for loop practice. The game loop takes whatever
+# mountain index is printed and chooses it as the mountain to fire on.
+#
+# *This section of code sets up the game loop*
+#
+# **Game Loop**
+import sys
+import math
+while True:
+    max = 0
+    maxIndex = -1
+    for i in range(8):
+        # We will use "mountain_h" as our variable for the height of a mountain
+        mountain_h = int(input())
+#
+# Now we must write an action using print:
+#
+# The index of the mountain to fire on in the first test case is 4. To fire on this mountain we use:
```

Comment:
Use the following syntax to show code examples:

```
# .. code-block:: Python
#   :linenos:
#
#   print(4)
#
```

Reply:
implemented throughout project. done